react/react-effects-and-fetch/react-effects-and-fetch_api-status/src/components/StatusCheck/index.jsx

```diff
@@ -20,6 +21,25 @@ export default function StatusCheck() {
      * is okay and false if it is not.
      **/
     // --v-- write your code here --v--
+    async function fetchData() {
```

Comment:
It's all working. But you didn't have to create a new asynchronous function and call it on line 42. It was enough just to add `async` to the already existing `handleCheckApiStatus()` function on line 10.